src/glass/src/app/shared/components/widget/widget.component.ts

```diff
@@ -62,14 +64,14 @@ export class WidgetComponent implements OnInit, OnDestroy {
         finalize(() => {
           this.error = !this.loadingWithoutError;
           this.firstLoadComplete = this.loadingWithoutError;
-          this.refreshDataSubscription?.unsubscribe();
           if (this.reloadTime > 0) {
-            setTimeout(() => {
-              this.reload();
-            }, this.reloadTime);
+            this.timerSubscription = timer(this.reloadTime)
+              .pipe(take(1))
```

Comment:
Maybe it's nicer to use [first()](https://rxjs-dev.firebaseapp.com/api/operators/first) here instead of `take(1)`.

Reply:
I wanted to use the `first` operator at first, but then i thought `take` expresses more the intention of this operator. What do you think?

Reply:
I thought `first` would be easier to understand, but you think the opposite. To not get into a bike shed discussion than - just leave it as it is ;)